tapfreighter/parcel.go

```diff
@@ -203,9 +203,9 @@ type PreSignedParcel struct {
 	// vPkt is the virtual transaction that should be delivered.
 	vPkt *tappsbt.VPacket
 
-	// inputCommitment is the commitment for the input that is being spent
-	// in the virtual transaction.
-	inputCommitment *commitment.TapCommitment
+	// inputCommitments are the commitments for the input that are being
+	// spent in the virtual transaction.
+	inputCommitments tappsbt.InputCommitments
```

Comment:
Ah so we didn't actually support multi-input here before?

Reply:
Yes, for the "externally signed" (vPSBT) flow we didn't fully implement the multi input everywhere, only for the address send. So there might be more commits needed to support some of the integration test cases I have in mind.